examples/evaluation/Evaluate_RAG_with_LlamaIndex.ipynb

```diff
@@ -0,0 +1,1090 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Fv9e2I1w-SLF"
+      },
+      "source": [
+        "**Stages within RAG**\n",
+        "\n",
+        "There are five key stages within RAG, which in turn will be a part of any larger application you build. These are:\n",
+        "\n",
+        "**Loading:** this refers to getting your data from where it lives – whether it’s text files, PDFs, another website, a database, or an API – into your pipeline. LlamaHub provides hundreds of connectors to choose from.\n",
+        "\n",
+        "**Indexing:** this means creating a data structure that allows for querying the data. For LLMs this nearly always means creating vector embeddings, numerical representations of the meaning of your data, as well as numerous other metadata strategies to make it easy to accurately find contextually relevant data.\n",
+        "\n",
+        "**Storing:** once your data is indexed you will almost always want to store your index, as well as other metadata, to avoid having to re-index it.\n",
```

Comment:
delete

Reply:
shall I delete the entire 'stages within RAG' section?

Reply:
oops I meant to highlight just "almost always" in the Storing bullet - just delete that

Reply:
I think you meant storing. Deleted it.